app/jobs/fs/download_form_related_documents_job.rb

```diff
@@ -4,5 +4,6 @@ def perform(fs_form, downloader: ::Fs::FormRelatedDocumentsDownloader)
 
     fs_form_downloader.download_related_document_by_type(:xsd)
     fs_form_downloader.download_related_document_by_type(:xslt_txt)
+    fs_form_downloader.download_related_document_by_type(:xslt_html)
```

Comment:
Skusal si spustit cely ten hlavny job, ktory ma postahovat ku vsetkym FS formularom toto XSLT? Zafunguje to ok, nevyfailuju joby? Existuje ku kazdemu formularu HTML XSLT? Ak nie, tak v robeni vizualizacie by sme mali dat aj txt alternativu, ak nie je mozne spravit HTML.